index.html

```diff
@@ -5007,13 +4983,259 @@ <h3 id="looping-AudioBufferSourceNode">
+                        <code>bufferTime</code> to <code>offset + ((currentTime
+                        - start) * computedPlaybackRate)</code>. This is the
+                        <a>playhead position</a> for the first output sample
+                        frame.
+                        </li>
+                        <li>Let <code>effectiveBufferTime</code> equal
+                        <code>bufferTime</code>.
+                        </li>
+                        <li>map the playhead position to the correct point
+                        within the loop body accounting for loop iterations:
+                          <ol>
+                            <li>While <code>loop</code> is true and
+                            <code>effectiveBufferTime</code> &gt;=
+                            <code>actualLoopEnd</code>,
+                              <ol>
+                                <li>Let <code>effectiveBufferTime</code> equal
+                                <code>effectiveBufferTime - (actualLoopEnd -
+                                actualLoopStart)</code>.
+                                </li>
+                              </ol>
+                            </li>
+                          </ol>
+                        </li>
+                        <li>Emit the result of
+                        <code>playbackSignal(effectiveBufferTime)</code> as the
+                        output frame at <code>index</code>.
+                        </li>
+                        <li>Increase <code>bufferTime</code> by <code>dt *
+                        computedPlaybackRate</code>.
+                        </li>
+                      </ol>
+                    </li>
+                    <li>Increase <code>index</code> by 1.
+                    </li>
+                    <li>Increase <code>currentTime</code> by <code>dt</code>.
+                    </li>
+                  </ol>
+                </li>
+                <li>If <code>currentTime</code> &gt;= <code>stop</code>,
+                consider playback for the node to have ended.
+                </li>
+              </ol>
+            </li>
+          </ol>
+          <p>
+            The following figures illustrate the behavior of the algorithm in
+            assorted key scenarios. Buffer optimization is not considered, but
+            by definition it does not materially affect the resulting playback.
+            In all figures, the following conventions apply:
+          </p>
+          <ul>
+            <li>context sample rate is 1000 Hz
+            </li>
+            <li>buffer samples are shown with the first sample frame at the
+            <em>x</em> origin.
+            </li>
+            <li>output samples are shown with the <code>start</code> context
+            time sample rate interpolation.
+            </li>
+          </ul>
+          <p>
+            Basic playback with a simple loop that ends after the last sample
```

Comment:
OK. I tried using these long phrases as captions but I felt they did not format very clearly, which is why the captions are now very brief and this text precedes each figure.